src/amp/index.rs

```diff
@@ -178,3 +178,116 @@ impl BTreeAmpIndex {
             .map_or_else(|| suffix.to_string(), |t| format!("{}{}", t, suffix))
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::amp::{
+        domain::{AmpIndexer, AmpResult, FullKeyword, OriginalAmp},
+        index::BTreeAmpIndex,
+    };
+
+    fn test_amp_fixture() -> OriginalAmp {
```

Comment:
this is probably really pytest-ish, is there a rust convention?

Reply:
This is the convention!